src/routes/+page.svelte

```diff
@@ -1,18 +1,26 @@
 <script lang="ts">
 	import { page } from '$app/stores';
-
 	import { supabase } from '$lib/auth';
 	import Link from '$lib/components/Link.svelte';
 	import Button from '$lib/components/Button.svelte';
 </script>
 
 <div class="container">
-	<h1>Welcome to Annuaire</h1>
-
-	<Link href="inscription">S'enregistrer</Link>
-	<br />
-	<Link href="connexion">Se connecter</Link>
-	<br />
+	<nav class="navbar">
+		<div class="logo">
+			<img src="../svg/logo.svg" alt="Logo" />
+		</div>
+		<ul class="navbar-tabs">
+			<li><a href="#">Accueil</a></li>
```

Comment:
Same here, doesnt match figma. You should add more space between logo and the first link : 

<img width="312" alt="Capture d’écran 2023-02-25 à 15 30 36" src="https://user-images.githubusercontent.com/46904617/221362439-da173d59-286e-49d5-9e03-6feddb9d1b29.png">